llvm/utils/TableGen/SubtargetEmitter.cpp

```diff
@@ -1781,13 +1781,68 @@ void SubtargetEmitter::EmitHwModeCheck(const std::string &ClassName,
   if (CGH.getNumModeIds() == 1)
     return;
 
-  OS << "unsigned " << ClassName << "::getHwMode() const {\n";
+  // Collect all HwModes and related features defined in the TD files,
+  // and store them in bit format.
```

Comment:
Similarly to earlier comments: "...store them in bit format." -> "store them as a bit vector."